### DISCUSSION TITLE
Can I use coordinator from an already supported device?

### DISCUSSION THREAD

**Discussion by hartungbrian** [Koenkk/zigbee2mqtt#19050] [Devices]
My new Frient Electricity Meter interface 2 is registered as "not supported". It's also brand new so fair enough.
It's more or less just a new version of develco zhemi101 which is supported.
https://community.home-assistant.io/t/frient-electricity-meter-develco-zhemi101-zigbee/386723

Is it possible to use the coordinator for zhemi101 on Frient Electricity Meter interface 2? If so, how? :-)
Thanks.

**Comment by AlmightyCZ** [comment]
Did you get it working?

**Reply by hartungbrian** [reply]
No unfortunately not :-(
I found a External converters in here but no data is coming in.
And I don't have the brain to figure out what I can do 🙈
